### PR TITLE
feat: created Store Screen

### PR DESCRIPTION
**Pull request by cr1spyvada**
- [x] Store Screen
- [x] Stock Recieve Entry
- [x] Stock Issue Entry
- [x] Phase to Phase Transfer
- [x] Branch to Branch Send
- [x] Branch to Branch Receive

### CODE REVIEW

File: apps/Dataman/src/_app.tsx [hunk 1]
```diff
@@ -1,10 +1,40 @@
...
+import { StoreScreen } from "./screens/store";
+import StockReceive from "./screens/stockRecieve";
+import StockIssue from "./screens/stockIssue";
+import PhasetoPhase from "./screens/phaseToPhase";
+import BranchSend from "./screens/branchSend";
+import BranchReceive from "./screens/branchReceive";
+
 export const App = () => {
+  const [fontsLoaded, setFontsLoaded] = React.useState(false);
+  React.useEffect(() => {
```
Comment (sutyum) [inline apps/Dataman/src/_app.tsx]: why do we need to load font in this manner? share reference

File: apps/Dataman/src/screens/branchReceive.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+import React from "react";
+import NavBar from "../components/NavBar";
+import { Button, List, Text } from "@ant-design/react-native";
+import { AntDesign } from "@expo/vector-icons";
+
+const BranchReceive = () => {
+  return (
+    <View>
+      <NavBar />
+      <ScrollView>
```
Comment (sutyum) [inline apps/Dataman/src/screens/branchReceive.tsx:43]: refactor this into an array containing list data and use .map() to iteratate over this list in order to create the list view

File: apps/Dataman/src/screens/branchSend.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+              style={{
+                fontSize: 28,
+                fontWeight: "600",
+                paddingVertical: 5,
+              }}
+            >
+              Branch to Branch Send
+            </Text>
+          </List.Item>
+          <List.Item arrow="horizontal">
```
Comment (sutyum) [inline apps/Dataman/src/screens/branchSend.tsx]: refactor this into an array containing list data and use .map() to iteratate over this list in order to create the list view

File: apps/Dataman/src/screens/phaseToPhase.tsx [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+            <Text
+              style={{
+                fontSize: 28,
+                fontWeight: "600",
+                paddingVertical: 5,
+              }}
+            >
+              Phase to Phase Transfer
+            </Text>
+          </List.Item>
```
Comment (sutyum) [inline apps/Dataman/src/screens/phaseToPhase.tsx:51]: refactor this into an array containing list data and use .map() to iteratate over this list in order to create the list view

File: apps/Dataman/src/screens/stockIssue.tsx [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+        <List style={{ marginBottom: 10 }}>
+          <List.Item>
+            <Text
+              style={{
+                fontSize: 28,
+                fontWeight: "600",
+                paddingVertical: 5,
+              }}
+            >
+              Stock Issue Entry
```
Comment (sutyum) [inline apps/Dataman/src/screens/stockIssue.tsx:37]: refactor this into an array containing list data and use .map() to iteratate over this list in order to create the list view

File: apps/Dataman/src/screens/stockRecieve.tsx [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+            <Text
+              style={{
+                fontSize: 28,
+                fontWeight: "600",
+                paddingVertical: 5,
+              }}
+            >
+              Stock Receive Entry
+            </Text>
+          </List.Item>
```
Comment (sutyum) [inline apps/Dataman/src/screens/stockRecieve.tsx:39]: refactor this into an array containing list data and use .map() to iteratate over this list in order to create the list view

File: apps/Dataman/src/screens/store.tsx [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+              {item}
+            </List.Item>
+          ))}
+        </List>
+      </View>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
```
Comment (sutyum) [inline apps/Dataman/src/screens/store.tsx]: why is this needed?

**Review by sutyum — CHANGES_REQUESTED** [review]

File: apps/Dataman/src/_app.tsx (hunk 1 above)
Comment (cr1spyvada) [reply apps/Dataman/src/_app.tsx]: It's mentioned in the docs that we have to load the fonts, while setting up the ant design library in the project.
https://rn.mobile.ant.design/docs/react/introduce
image

**Review by sutyum — APPROVED** [review]

**Pull request merged**